smalltalksrc/Slang/TMethod.class.st

```diff
@@ -916,6 +925,15 @@ TMethod >> ensureConditionalAssignmentsAreTransformedIn: aCodeGen [
 	 and: [self replaceNodesIn: transformedAssignments.
 		  true]] whileTrue
 		
+]
+
+{ #category : 'inlining' }
+TMethod >> ensureIsValidMethodAfterInlining: aNode in: aCodeGen [
+	"inlining can cause some issues in the AST that will produce incorrect code, modify the AST if needed to produce a cleaner one. Return true if anything was done "
+	^ self iterateThrought: aNode in: aCodeGen 
```

Comment:
We can get a better method name in here! First, there is a small typo :) => `iterateThrough`.

But also, the method does not *iterate*, it does things *while it iterates*. The method should say what it actually does!